### PR TITLE
Fix usage of System.currentTimeMillis().

### PR DESCRIPTION
**Pull request by bmarty**
Should make the CI happy.

### CODE REVIEW

File: library/core-utils/src/main/java/im/vector/lib/core/utils/flow/TimingOperators.kt [hunk 1]
```diff
@@ -68,10 +69,10 @@ fun <T> Flow<T>.chunk(durationInMillis: Long): Flow<List<T>> {
 
 @ExperimentalCoroutinesApi
 fun <T> Flow<T>.throttleFirst(windowDuration: Long): Flow<T> = flow {
-    var windowStartTime = System.currentTimeMillis()
+    var windowStartTime = SystemClock.elapsedRealtime()
```
Comment (ouchadam) [inline library/core-utils/src/main/java/im/vector/lib/core/utils/flow/TimingOperators.kt:72]: ideally we'd make use of the `Clock` here as directly interacting with system isn't too test friendly (and have the `Clock` as part of `core`), the but I understand this is a quick workaround to get the CI green again

**Review by ouchadam — APPROVED** [review]

**Review by onurays — CHANGES_REQUESTED** [review]
Other usage in `DefaultClock` class is also failing. Can you fix it too?

**Comment by ouchadam** [comment]
@onurays the usage in `DefaultClock` should be allowed via the 2 usages threshold https://github.com/vector-im/element-android/blob/develop/tools/check/forbidden_strings_in_code.txt#L179

unfortunately when the script fails it prints out the first usage it finds, which tends to be the allowed usage :cry:

**Comment by onurays** [comment]
> @onurays the usage in `DefaultClock` should be allowed via the 2 usages threshold https://github.com/vector-im/element-android/blob/develop/tools/check/forbidden_strings_in_code.txt#L179
> 
> unfortunately when the script fails it prints out the first usage it finds, which tends to be the allowed usage 😢

Oh, right. So the purpose is to use Clock everywhere, but difficult to inject here.

**Review by onurays — APPROVED** [review]
Aggree on @ouchadam's comment. Let's make the CI happy again.

File: library/core-utils/src/main/java/im/vector/lib/core/utils/flow/TimingOperators.kt (hunk 1 above)
Comment (bmarty) [reply library/core-utils/src/main/java/im/vector/lib/core/utils/flow/TimingOperators.kt:72]: I agree, yes, quick fix for now.
Also will be a bit painful to pass a `Clock` to every `throttleFirst` call.

**Pull request merged**